data/os/OpenBSD.yaml

```
@@ -6,5 +6,6 @@ nsd::service_name: 'nsd'
 nsd::owner: '_nsd'
 nsd::group: '_nsd'
 nsd::control_cmd: 'nsd-control'
+nsd::control_setup_cmd: 'nsd-control-setup'
```

These seem to be part of the base system:
```suggestion
nsd::control_cmd: '/usr/sbin/nsd-control'
nsd::control_setup_cmd: '/usr/sbin/nsd-control-setup'
```